src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryTable.cs

```diff
@@ -212,6 +212,20 @@ public static FormulaValue IndexTable(IRContext irContext, FormulaValue[] args)
             return arg0.Index(rowIndex).ToFormulaValue();
         }
 
+        public static async ValueTask<FormulaValue> Shuffle(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, FormulaValue[] args)
```

Comment:
>EvalVisitor runner, SymbolContext symbolContext,  [](http://example.com/codeflow?start=60&length=49)

The signature here could be simpler (see IndexTable above). 
It doesn't need to be async; and doesn't need runner object (since it doesn't invoke lambdas). So it could just be:

FormulaValue Shuffle(IRContext irContext, FormulaValue[] args)

There are some overloads in the library setup that allow various signatures.

Reply:
Thanks, i addressed this change